cli_scripts/voice_coding_cli.py

```diff
@@ -0,0 +1,26 @@
+import subprocess,sys,click
+
+@click.command()
+@click.option('--voice/--no-voice', default = False, help = "Use --voice if you want to code using your voice.")
```

Comment:
I might be missing something, but don't you always want to code using your voice?

Reply:
We kept that as an option in case we want to add more features, if nothing comes up, we'll strip --no-voice.